### DISCUSSION TITLE
Disconnected. Retrying in X seconds ...

### DISCUSSION THREAD

**Discussion by sairaamv** [h2oai/wave#1236] [Q&A] [answered]
Is there a way to avoid disconnect with the wave server? 
Do we have any time-limits if the client is not active?

Thanks,
Sai

**Comment by lo5** [comment, marked as answer]
There is no time limit. The front-end will retry until the server is up, and then bring itself back to a consistent state.

> Is there a way to avoid disconnect with the wave server?

Not sure what you mean. If the browser is disconnected from the server (either flaky internet connection or server died), then the UI will say it's disconnected (because you might be unknowingly viewing stale information otherwise).
